### PR TITLE
UT

### PR DESCRIPTION
**Pull request by sayan-rudder**
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/82)

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  coverageDirectory: "coverage",
+
+  // An array of regexp pattern strings used to skip coverage collection
+  coveragePathIgnorePatterns: [
+    "/node_modules/",
+    "/dist/",
+    "/tests/",
+    "/scripts/",
+    "__tests__"
+  ],
```
Comment (arnabp92) [inline jest.config.js:33]: coverage should exclude test files as well? `*.test.js` ?

File: package.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
     }
   ],
   "scripts": {
-    "test": "./scripts/run_tests.sh",
-    "prodTest": "npm run buildProdBrowser && sed 's/var rudderanalytics/rudderanalytics/g' dist/rudder-analytics.min.js > __tests__/prod_test.js && jest script.test.js",
+    "test": "jest --coverage",
```
Comment (arnabp92) [inline package.json:14]: Can we add `--notify` as well? Good to have!

File: jest.config.js (hunk 1 above)
Comment (sayan-rudder) [reply jest.config.js:33]: no..this is for the folder which will has other js files as well, so don't want that all files in folder.
the key which controls the test file names is **testMatch**

File: package.json (hunk 2 above)
Comment (sayan-rudder) [reply package.json:14]: okay

**Pull request merged**